docs/source/doc.gptable.rst

```diff
@@ -135,6 +135,179 @@ This ``additional_formatting`` parameter is best demonstrated by example:
+options for `numeric formats in the XlsxWriter documentation`_.
+
+.. _`numeric formats in the XlsxWriter documentation`: https://xlsxwriter.readthedocs.io/format.html#format-set-num-format
+
+.. list-table::
+    :header-rows: 1
+    :widths: 19 30
+    :align: left
+
+    * - Description
+      - Example usage
+    * - Numeric format
+      - | {"num_format": 1} # Format index 
+        | {"num_format": "d mmm yyyy"} # Format string
+.. 
+
+-----------
+Protection formatting
+-----------
+
+This table demonstrates the protection methods available.
+
+.. list-table::
+    :header-rows: 1
+    :widths: 19 30
+    :align: left
+
+    * - Description
+      - Example usage
+    * - Lock cells
+      - {"locked": True}
+    * - Hide formulas
+      - {"hidden": True}
+.. 
+
+-----------
+Alignment formatting
+-----------
+
+This table demonstrates the alignment formatting options available. You can find all options for 
+`horizontal and vertical alignment in the XlsxWriter documentation`_.
+
+.. _`horizontal and vertical alignment in the XlsxWriter documentation`: https://xlsxwriter.readthedocs.io/format.html#format-set-align
+
+.. list-table::
+    :header-rows: 1
+    :widths: 19 30
+    :align: left
+
+    * - Description
+      - Example usage
+    * - Horizontal align
+      - {"align": "center"}
+    * - Vertical align
+      - {"align": "vcenter"}
+    * - Rotation
+      - {"rotation": 30}
+    * - Text wrap
+      - {"text_wrap": True}
+    * - Center across
+      - {"set_center_across": True}
+    * - Indentation
+      - {"indentation":2}
+    * - Shrink to fit
+      - {"shrink": True}
+.. 
+
+-----------
+Pattern formatting
+-----------
+
+This table demonstrates the pattern formatting options available.
+
+.. list-table::
+    :header-rows: 1
+    :widths: 19 30
+    :align: left
+
+    * - Description
+      - Example usage  
+    * - Cell pattern
+      - {"pattern": 1}
+    * - Background colour
+      - {"bg_color": "white"}
+    * - Foreground colour
+      - {"fg_color": "white"}
+.. 
+
+-------------
+Border formatting
+-------------
+
+This table demonstrates the border formatting options available. You can find all options 
+for `border styles in the XlsxWriter documentation`_.
+
+.. _`border styles in the XlsxWriter documentation`: https://xlsxwriter.readthedocs.io/format.html#format-set-border
+
+.. list-table::
+    :header-rows: 1
+    :widths: 19 30
+    :align: left
+
+    * - Description
+      - Example usage
+    * - Cell border 
+      - {"border": 1}
+    * - Bottom border
+      - {"bottom": 1}
+    * - Top border
+      - {"top": 1}
+    * - Left border
+      - {"left": 1} 
+    * - Right border
+      - {"right": 1}
+    * - Border colour
+      - {"border_color": "red"}
+    * - Bottom colour
+      - {"bottom_color":"red"}
+    * - Top colour
+      - {"top_color": "red"}
+    * - Left colour
+      - {"left_color": "red"}
+    * - Right colour
+      - {"right_color": "red"}
```

Comment:
Consider varying the the value here to demonstrate different ways to specify colour, eg hex code etc